pkg/sypgp/sypgp.go

```diff
@@ -285,7 +285,7 @@ func StorePubKey(e *openpgp.Entity) (err error) {
 // key and oldToken match.
 func compareLocalPubKey(e *openpgp.Entity, oldToken string) bool {
```

Comment:
uppercase this function name and update the reference where needed

Reply:
Is this something that we plan on using outside of this package? Should only uppercase things we want to be part of the exported API of the package.

Reply:
I changed the function name to `CompareKeyEntity()`, hopefully everyone is happy :smile:

Reply:
@ikaneshiro gotcha, well if it's only intended to be used on this package of course it makes no sense to uppercase it :+1: